src/vms/reward-vm.js

```diff
@@ -9,8 +9,9 @@ const error = m.prop(''),
     states = m.prop([]),
     fees = m.prop([]),
     noReward = {
-        id: -1,
```

Comment:
Are you sure we don't use this value as `-1` for any reasons on other components?

Reply:
Apparently not. This hash is overwritten later on anyway.